src/main.cpp

```diff
@@ -197,6 +197,9 @@ int main (int argc, char* argv[128]) {
         else if (!strcmp(argv[i], "-q")) {
             quiet = true;
         }
+        else if (!strcmp(argv[i], "-r") || !strcmp(argv[i], "--reduce")) {
```

Comment:
Need to update help message - see `print_usage_and_exit()` above.

Reply:
Updated according to your comment.